pool.go

```diff
@@ -401,8 +432,10 @@ func (p *pool) doFetch(ctx context.Context, from string, c cid.Cid, attempt int)
 	resp, err := p.config.SaturnClient.Do(req)
 	fb = time.Now()
```

Comment:
This records the time when headers arrive, right? Let's instead record the time when the first bytes of the body arrive. That's how the other retrieval clients measure TTFB and it's harder for the L1 to cheat that.

Reply:
Done.